AdyenCard/Utilities/BinLookup/BinLookupService.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Provide cardType detection based on BinLookup API.
+internal protocol AnyBinLookupService {
```

Comment:
can we call it `Provider` like the `CardPublicKeyProvider`? or call the `CardPublicKeyProvider` `CardPublicKeyService`? for consistency?

Reply:
Yep, I thought about it.. `BinLookupProvider` is ok